src/reactjs/tests/useQuery.test.tsx

```diff
@@ -4616,7 +4616,7 @@ describe('useQuery', () => {
 
     expect(states[0]).toMatchObject({
       data: undefined,
-      isLoading: true,
+      isLoading: false,
```

Comment:
This test in current master (version 3.x.x) has `isLoading: false`:
https://github.com/tannerlinsley/react-query/blob/09a375b/src/react/tests/useQuery.test.tsx#L4494

But seems like this behaviour was missed during `isIdle` refactoring in this PR:
https://github.com/tannerlinsley/react-query/pull/3302/files#diff-eee051dcfac05e804da529ab297862d9341cf6d3963d5fe0c2b3ed1918ca5decR4452